### PR TITLE
Refactoring the README

### PR DESCRIPTION
**Pull request by nuwand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,23 @@
-#WSO2 API Management Server
+# WSO2 API Management
 
+WSO2 API Management is an Open Source API Management tool that allows APIs/Services to be Secured, Monitored and Rate Controlled. This tool allows API Developers to design, pubilsh, version and manage API lifecycles. It allows application developers to discover and start consuming APIs. The lightweight API Gateway intercepts all API requests applying Security, Rate Limiting and Monitoring aspects on your APIs.
```
Comment (praminda) [inline README.md]: Spelling mistake 

> This tool allows API Developers to design, **pubilsh**

Comment (praminda) [inline README.md]: Spelling mistake 

> This tool allows API Developers to design, **pubilsh**

File: README.md [hunk 2]
```diff
@@ -1,30 +1,23 @@
...
-@Path ("/passthrough")
-resource passthrough (message m) {
-   reply invoke(stockEP, m);
-}
-```
+1. Download the WSO2 Identity Server from [here](http://wso2.com/identity-and-access-management#download)
+2. Extract the downloaded wso2is-5.3.0.zip file
+3. Go to the wso2is-5.3.0/bin directory and execute the wso2server.sh script using the command ``./wso2server.sh start``
+4. Find the API Manager product distribution from product-apim/product/target
+5. Extract the distribution. Ex: unzip wso2apim-3.0.0.zip
```
Comment (prasa7) [inline README.md]: Are we putting the version as wso2apim-3.0.0.zip or do we need to change this to wso2apim-3.0.0-SNAPSHOT.zip until we release 3.0.0 ?

Comment (nuwand) [inline README.md]: Let's leave it as 3.0.0.zip. I put it as an example so that we don't have to be explicit.

File: README.md (hunk 1 above)
Comment (nuwand) [inline README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (nuwand) [inline README.md]: Let's leave it as 3.0.0.zip, I put it as an example so that we don't have to be explicit.

**Pull request merged**